src/imitation/policies/base.py

```diff
@@ -63,14 +64,30 @@ class FeedForward32Policy(policies.ActorCriticPolicy):
 
     Note: This differs from stable_baselines3 ActorCriticPolicy in two ways: by
     having 32 rather than 64 units, and by having policy and value networks
-    share weights except at the final layer.
```

Comment:
do they share weights at the final layer?

Reply:
- SB3 [ActorCriticPolicy](https://github.com/DLR-RM/stable-baselines3/blob/d68f0a2411766beb6da58ee0e989d1a6a72869bc/stable_baselines3/common/policies.py#L455) uses `[dict(pi=[64, 64], vf=[64, 64])]` as its default network.
- According to [Custom Policy Network in SB3](https://stable-baselines3.readthedocs.io/en/master/guide/custom_policy.html#on-policy-algorithms), they don't share any layer.

Reply:
So if I understand correctly:
  - SB3 does no weight sharing by default.
  - We do weight sharing (except at the final layer, where there's a different linear head).
This seems consistent with the original comment, but I can see that it's a bit ambiguous, but perhaps clarify rather than remove the "except at the final layer" caveat.

Reply:
Oh I see what you mean. You're correct that it would be better to specify the different linear heads.